frontend/src/components/chart_panel.jsx

```diff
@@ -102,7 +102,13 @@ const ChartPanel = ({ settings, id, setPanels }) => {
             let seriesData = new Array();
             param.seriesData.forEach((data, key, map) => {
                 if (!key.options().visible) return;
-                let color = key.options().color || key.options().topFillColor1 || key.options().lineColor || key.options().upColor || key.options().borderColor;
+                let color = [
```

Comment:
@eliza-eliza my bad. I think you were the first time but formatting threw me off.